pkg/synopsysctl/cmd_destroy.go

```diff
@@ -52,6 +52,7 @@ var destroyCmd = &cobra.Command{
 		// Read Commandline Parameters
 		if len(args) > 0 {
 			for _, operatorNamespace := range args {
+				log.Infof("destroying Synopsys Operator in namespace '%s'...", operatorNamespace)
 				return destroy(operatorNamespace)
 			}
 		} else {
```

Comment:
We should only act if there is 1 operator in the system and it is cluster scoped.  Anything else is ambiguous to the user.

Reply:
The customer can have 3 cluster scope operators with one resource enabled on each operator.

Reply:
True, but even more so why we shouldn't act if there isn't a namespace given. :) In the case where the user just does "synopsysctl destroy" we need to either error all the time unless a NS is given, or only act if there is 1 operator in the system.  There's no other way the user could know what to expect.

Reply:
done